### ISSUE TITLE
Is this repo reasonably forkable?

### ISSUE THREAD

**Issue by gs-rezaem** [AdoptOpenJDK/openjdk-jdk8u#5]
We forked this repo a few days ago (https://github.com/goldmansachs/openjdk-jdk8u) .
The top commit on the master branch was bd081d9 (https://github.com/goldmansachs/openjdk-jdk8u/commit/bd081d9c771a625f7b621d209e9324f1f6a5a620).

That commit has been removed and replaced with another commit c8f8a60 (https://github.com/AdoptOpenJDK/openjdk-jdk8u/commit/c8f8a609bc36c7939e5d65d39aaac35e39ce3b60).

In other words, the history of this branch was re-written.

That makes forking this repo and keeping the fork up to date quite hard, as updates cannot be fast forward merged and rebase causes conflicts.

Is this expected, or was the commit in error?

**Comment by gdams** [comment]
there was a few force pushes required to set up the new merge job, it shouldn't be an issue anymore, I apologise for the inconvenience

**Comment by gs-rezaem** [comment]
no problem. thanks.
